airbyte-ci/connectors/pipelines/README.md

```diff
@@ -398,6 +398,7 @@ This command runs the Python tests for a airbyte-ci poetry package.
 ## Changelog
 | Version | PR                                                         | Description                                                                                               |
 | ------- | ---------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| 2.2.5   | [#31716](https://github.com/airbytehq/airbyte/pull/31716)  | Enable pre-release publish with local CDK.
```

Comment:
```suggestion
| 2.3.0   | [#31716](https://github.com/airbytehq/airbyte/pull/31716)  | Enable pre-release publish with local CDK.
```

Isn't it a new feature 😄 ?

Reply:
I guess so - felt more like a bug fix as I just had to pass along something that was getting dropped 😄